File: cli/dsound.py
#!/usr/bin/env python3
"""Back-compat entry. Real commands live in cli/main.py."""

from pathlib import Path
import sys

sys.path.insert(0, str(Path(__file__).resolve().parent))
from main import main

if __name__ == "__main__":
    raise SystemExit(main())

File: bin/dsound.js
#!/usr/bin/env node
/**
 * Thin wrapper. No commander. No fake ESS wallet.
 * Real work: python cli/main.py
 */
const { spawnSync } = require("child_process");
const path = require("path");

const root = path.resolve(__dirname, "..");
const args = process.argv.slice(2);
const result = spawnSync("python3", [path.join(root, "cli", "main.py"), ...args], {
  cwd: root,
  stdio: "inherit",
});
process.exit(result.status === null ? 1 : result.status);
